src/data/faq_de.json

```
@@ -251,7 +253,8 @@
                         "active": true,
                         "textblock": [
                             "Die App ist in den folgenden Sprachen verfügbar:<ul><li>Deutsch</li><li>Englisch</li><li>Türkisch (ab Version 1.1.1)</li><li>Bulgarisch (ab Version 1.2.0)</li><li>Polnisch (ab Version 1.2.0)</li><li>Rumänisch (ab Version 1.2.0)</li></ul>",
-                            "Die Hotline (<a href='tel:+498007540001'>+49 800 7540001</a>) wird auf Deutsch, Englisch und Türkisch bedient. Weitere Sprachen sind geplant. Wir halten Sie auf dem Laufenden.", "Die Sprache der App orientiert sich an der Sprache, die in den Systemeinstellungen Ihres Smartphones eingestellt ist. Um die App etwa auf Englisch anzuzeigen, ändern Sie die Sprache in den Einstellungen Ihres Smartphones auf Englisch."
+                            "Die Hotline (<a href='tel:+498007540001'>+49 800 7540001</a>) wird auf Deutsch, Englisch und Türkisch bedient. Weitere Sprachen sind geplant. Wir halten Sie auf dem Laufenden.", "Die Sprache der App orientiert sich an der Sprache, die in den Systemeinstellungen Ihres Smartphones eingestellt ist. Um die App etwa auf Englisch anzuzeigen, ändern Sie die Sprache in den Einstellungen Ihres Smartphones auf Englisch.",
+                            "Bitte beachten Sie, dass viele Länder eigene Corona-Apps haben. Weil die deutsche Corona-Warn-App auch in den App Stores und Play Stores dieser Länder verfügbar ist, könnten die Corona-Apps verwechselt werden. Achten Sie daher darauf, dass Sie die App des Landes, in dem Sie sich aufhalten, herunterladen."
```

```suggestion
                            "Bitte beachten Sie, dass viele Länder eigene Corona-Apps haben. Weil die deutsche Corona-Warn-App auch in den App Stores und Play Stores dieser Länder verfügbar ist, könnten die Corona-Apps verwechselt werden. Achten Sie daher darauf, dass Sie die App Ihres Landes herunterladen."

```
Nicht "des Landes, in dem Sie sich aufhalten". Sonst müssen wir hier noch ins Detail gehen, wie das mit dem Wechsel zwischen landesspezifischen Apps funktioniert. Für mich ist das aber eine andere Fragestellung und sollte nicht hier behandelt werden.